src/lamplib/src/genny/cli.py

```diff
@@ -430,6 +418,15 @@ def canaries(ctx: click.Context, canary_args: List[str]):
         "Not needed if you have built/installed these tools in hte --mongo-dir."
     ),
 )
+@click.option(
+    "--mongodb-commit",
+    type=str,
+    required=False,
+    default=None,
+    help=(
+        "The commit corresponding to the mongodb binary passed with -mongodb-archive-url option."
+    ),
+)
```

Comment:
I'm not 100% convinced that the `--mongodb-commit` flag is necessary. I think someone who needs it could just clone the repo, check out whatever commit they want, and specify that dir. 

But won't block the PR on that, up to you either way!

Reply:
I agree. Removing the option